### DISCUSSION TITLE
Trying to make a live bot on Docker with AntMedia

### DISCUSSION THREAD

**Discussion by Skam94** [ant-media/Ant-Media-Server#6193] [FAQs]
I build a composer running the latest Ubuntu version and installed ant media i contacted the customer service and they telled men no need to have a liscence but when i am trying to run this commande 

`cd /usr/local/antmedia/ && sudo ./enable_ssl.sh`

i get this message

`Please make sure you enter your license key and use the Enterprise edition.`

**Comment by yashtandon113** [comment]
@Skam94 
Please make sure that you have installed the Ant Media Server Enterprise Edition on your Docker container. Because the free domain option works with the Enterprise edition, you need a license key.

If you do not have EE license yet, you can subscribe to free trial at https://antmedia.io/freetrial/

Please feel free to ask if you have any questions.
